### PR TITLE
Added info for Inertia.js

### PR DESCRIPTION
**Pull request by jxxe**
- It's written Inertia.js, not Inertia like the logo
- Laravel or Ruby on Rails are implied by Inertia.js
- The confidence numbers come from the [Laravel](https://github.com/inertiajs/inertia-laravel/network/dependents) and [Rails](https://github.com/inertiajs/inertia-rails/network/dependents) repositories (16,438 vs 128 uses)
- It also implies React, Vue, or Svelte but I think those can be detected better other ways

### CODE REVIEW

File: src/technologies/i.json [hunk 1]
```diff
@@ -501,6 +501,10 @@
     "headers": {
       "X-Inertia": ""
     },
+    "implies": [
+      "Laravel\\;confidence:99",
+      "Ruby on Rails\\;confidence:1"
```
Comment (AliasIO) [inline src/technologies/i.json:506]: Implies should not have `confidence`, unless Inertia.js always requires Laravel or RoR, these should be removed.

Comment (jxxe) [reply src/technologies/i.json:506]: Inertia.js is a tool that makes it easier to build frontends for Laravel and Ruby on Rails apps, so it always requires one of them.

Comment (AliasIO) [reply src/technologies/i.json:506]: The `implies` means it will always positively detect both 'Laravel' and 'Ruby on Rails' here. I'll remove these.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**